### PR TITLE
fix: upgrade IOS bindings to firebase 8.x

### PR DESCRIPTION
**Pull request by mk-5**
fix for #63

### CODE REVIEW

File: gdx-fireapp-ios/src/pl/mk5/gdx/fireapp/ios/analytics/Analytics.java [hunk 1]
```diff
@@ -64,12 +64,7 @@ public void logEvent(String name, Map<String, String> params) {
      */
     @Override
     public void setScreen(final String name, final Class<?> screenClass) {
-        NSOperationQueue.getMainQueue().addOperation(new Runnable() {
-            @Override
-            public void run() {
-                FIRAnalytics.setScreenName(name, screenClass.getSimpleName());
```
Comment (prateekchandan) [inline gdx-fireapp-ios/src/pl/mk5/gdx/fireapp/ios/analytics/Analytics.java:70]: Should we remove this API altogether from the Core Analytics file ?

File: gradle/dependencies.gradle [hunk 2]
```diff
@@ -14,7 +14,7 @@ ext {
     spockVersion = '1.2-groovy-2.4'
     groovyVersion = '2.4.15'
     byteBuddyVersion = '1.8.21'
-    altPodsVersion = '1.16.0-SNAPSHOT'
+    altPodsVersion = '1.17.0-SNAPSHOT'
```
Comment (prateekchandan) [inline gradle/dependencies.gradle:17]: I am using 1.26.0-SNAPSHOT in my app. where all google signing fails.

File: e2e/core/src/pl/mk5/gdx/fireapp/e2e/GdxFireappE2ETests.java [hunk 3]
```diff
@@ -55,8 +56,8 @@ public GdxFireappE2ETests() {
         e2ETestRunner.addNext(new AuthAnonymousTest());
         e2ETestRunner.addNext(new AuthCreateUserEmailPasswordTest());
         e2ETestRunner.addNext(new AuthSignInUserEmailPasswordTest());
-//        e2ETestRunner.addNext(AuthGoogleSignInTest.class);
-        e2ETestRunner.addNext(new AuthAppleSignInTest(), 360);
+//        e2ETestRunner.addNext(new AuthGoogleSignInTest(), 360);
```
Comment (prateekchandan) [inline e2e/core/src/pl/mk5/gdx/fireapp/e2e/GdxFireappE2ETests.java:59]: Why are these e2e tests failing ?

Comment (mk-5) [reply e2e/core/src/pl/mk5/gdx/fireapp/e2e/GdxFireappE2ETests.java:59]: they are not failing / but they required user interaction. So they cannot be run from ci. They need to be run manually

File: gradle/dependencies.gradle (hunk 2 above)
Comment (mk-5) [reply gradle/dependencies.gradle:17]: 1.26.0-SNAPSHOT is "to new". Google sign-in api had some breaking changes, and sign-in flow works a bit different. There is no way to simple bump version to 1.26. More changes in the code are required

File: gdx-fireapp-ios/src/pl/mk5/gdx/fireapp/ios/analytics/Analytics.java (hunk 1 above)
Comment (mk-5) [reply gdx-fireapp-ios/src/pl/mk5/gdx/fireapp/ios/analytics/Analytics.java:70]: it's still working for android, and others I guess 🤔

**Pull request merged**